### ISSUE TITLE
Crate not found for alternate Cargo.toml path syntax

### ISSUE THREAD

**Issue by TheNewJavaman** [intellij-rust/intellij-rust#9409]
## Environment

* **IntelliJ Rust plugin version:** 0.4.179.4900-222
* **Rust toolchain version:** 1.66.0-nightly (e7119a030 2022-09-22) x86_64-pc-windows-msvc
* **IDE name and version:** IntelliJ IDEA 2022.2.2 Ultimate Edition (IU-222.4167.29)
* **Operating system:** Windows 11 10.0
* **Macro expansion:** enabled

## Problem description

There are two general syntaxes for including path dependencies in `Cargo.toml`. When the local crate exists, one syntax does not give a warning (which is correct behavior), while one does give a warning: `Crate [_] not found`. Example:

```toml
my-local-crate.path = "my-local-crate" # Gives warning (incorrect)
my-local-crate = { path = "my-local-crate" } # Does not give warning (correct)
```

The correct behavior should be no error.

## Screenshots

The example below uses a local workspace in one of my personal projects.

<img width="279" alt="image" src="https://user-images.githubusercontent.com/16387264/192176024-e709e076-350b-4c3a-862c-eba5de3ee3c6.png">

<img width="262" alt="image" src="https://user-images.githubusercontent.com/16387264/192176066-7d6e7c36-b24b-4275-94f5-8cb7244ddc29.png">

I hope this helps 🙂

**Comment by TheNewJavaman** [comment]
To clarify, the project builds fine. It's just the inspection that's incorrect.
